cerberus/kubernetes/client.py

```diff
@@ -90,3 +92,27 @@ def monitor_component(iteration, component_namespace):
     logging.info("Iteration %s: %s: %s"
                  % (iteration, component_namespace, watch_component_status))
     return watch_component_status, failed_component_pods
+
+
+# Load env variables and initialize slack python client
+def initialize_slack_client():
```

Comment:
Think it's better to create a separate module for slack integration. Thoughts?

Reply:
Yeah, that's better.